opencog/nlp/sureal/SuRealPMCB.cc

```diff
@@ -130,14 +131,15 @@ bool SuRealPMCB::variable_match(Handle &hPat, Handle &hSoln)
         // loop thru all connectors on both list
         while (not sourceConns.empty() && not targetConns.empty())
         {
-            // if we are repeating a multi-connector from source list
+            bool bResult;
+
             if (hMultiConn != Handle::UNDEFINED)
-                scmCode = "(lg-conn-linkable? " + SchemeSmob::to_string(hMultiConn) + " " + SchemeSmob::to_string(targetConns.front()) + ")";
+                bResult = lg_conn_linkable(hMultiConn, targetConns.front());
```

Comment:
> I assume that you are replacing code in order to make it run faster?

Yes, that is why.  Since this code is called for each connector on each disjunct on each node match, there could be several thousands scm call per node matches.  This change did make a difference.